packages/core/src/browser/window/default-secondary-window-service.ts

```diff
@@ -124,6 +135,48 @@ export class DefaultSecondaryWindowService implements SecondaryWindowService {
         return newWindow;
     }
 
+    protected findSecondaryWindowCoordinates(widget: ExtractableWidget): (number | undefined)[] {
+        const clientBounds = widget.node.getBoundingClientRect();
+        const preference = this.preferenceService.get('window.secondaryWindowPlacement');
+
+        let height; let width; let left; let top;
+        const offsetY = 20;
+
+        switch (preference) {
+            case 'originalSize': {
+                height = widget.node.clientHeight;
+                width = widget.node.clientWidth;
+                left = window.screenLeft + clientBounds.x;
+                top = window.screenTop + (window.outerHeight - window.innerHeight) + offsetY;
+                if (environment.electron.is()) {
+                    top = window.screenTop + clientBounds.y;
+                }
+                break;
+            }
+            case 'halfSize': {
+                height = window.innerHeight - (window.outerHeight - window.innerHeight);
```

Comment:
From the preference name I would have expected the height to be half the size as well. 
Should we adjust the preference name/height computation/preference description?

Reply:
Good point, I changed it to `halfWidth`. I expect users to use this preference like a split-screen mode.